client/app/stores/message_store.coffee

```diff
@@ -49,7 +49,7 @@ class MessageStore extends Store
             message.date          ?= new Date().toISOString()
             message.createdAt     ?= message.date
             message.flags         ?= []
-            message.hasAttachments = message.attachments.length > 0
+            message.hasAttachments = message.attachments.size > 0
```

Comment:
@m4dz: a bug should appear according to `line48` : if no attachement this will crash.